applications/error-notifications.md

```diff
@@ -0,0 +1,32 @@
+# Error Notifications
+
+In order to alert teams to server errors for applications we need a way of
+posting these out from applications.  Currently we have development and
+production instances of [Errbit](https://github.com/errbit/errbit), an
+open source product compatible with [airbrake](https://airbrake.io/)
+
+Details of these instances can be obtained from your local development lead.
+
+## NodeJS
+
+[node-airbrake](https://github.com/airbrake/node-airbrake) can be used for Node
+applications.  It has recently had an update for compatibility with HapiJS
+(it already had expressJS), however in order to load it correctly as a plugin
+or using glue and a manifest please see the HapiJS section below.
+
+### HapiJS
```

Comment:
Thanks, @teddmason you have answered it. If it helps my brain was coming from would you need **node-airbrake** for all errors raised outside of Hapi, and **node-hapi-airbrake** for anything raised by Hapi. Clearly you don't!

So I think just an opening line would do the trick

> ### HapiJS
> 
> If your application is using [Hapi](http://hapijs.com/) you should use [node-hapi-airbrake](https://github.com/EnvironmentAgency/node-hapi-airbrake) instead. **node-hapi-airbrake** is a ...